app/src/main/res/color/button_selector.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:state_checked="true" android:color="@color/white" />
-    <item  android:color="@color/colorControlNormal" />
+    <item android:state_checked="true" android:color="#FFFFFFFF" />
```

Comment:
Why no longer using the color references?

Reply:
I actually forgot to take a further step on this. I'm creating two specific colours for these, so it's more understandable... Something like radioChecked and radioUnchecked... Makes sense?

Reply:
PS: I'm not using @color/white because it's in RGB, not ARGB, being alpha the important part in these two colours.

Reply:
But isn't `#FFFFFF` equivalent with `#FFFFFFFF` (as in, the `FF` opacity is implicit)? Or am I missing something?

Reply:
Yes, sure. But if the whole point is the transparency/lack of it, relying on an implicit lack of transparency is way less readable than making it explicit it, right?

Reply:
I think that it's quite a standard on the web and on Android as well that 3-byte RGB values imply 100% opacity, but I might be wrong here. 
Anyways, since you used other color resource names, I am happy.